Service definition for ccp

akscram opened the pull request.

akscram committed:
Add initial definition for ccp

akscram committed:
Use find_packages to properly define package

akscram committed:
Fail if configuration is not valid

maretskiy commented on availability/tests/unit/test_config.py:
```diff
@@ -16,7 +16,7 @@
 
 from availability import config
-from tests.unit import test
+from availability.tests.unit import test
```
Tests module is something separate, it is not a part of availability module. I would request keeping tests module in old place (package root)

akscram committed:
Split tests on separate test cases

It is hard to add additional tests in a chain of resets because you have
to override return_value and side_effect attributes.

More accurately handle global single to return it in the original state
and do not dirty it for other tests.

akscram committed:
Add validation of the default config
